src/plainer.ts

```diff
@@ -216,6 +216,14 @@ export const walker = (
   const innerAnnotations: Record<string, Tree<TypeAnnotation>> = {};
 
   forEach(transformed, (value, index) => {
+    if (
+      index === '__proto__' ||
+      index === 'constructor' ||
+      index === 'prototype'
+    ) {
+      return;
```

Comment:
```suggestion
      throw new Error(`Detected property "`${index}". This is a prototype pollution risk, please remove it from your object.`)
```

Something like this. Ideally, we'd include the path to the malicious property in the error message, so it's easier to debug.